src/Components/People.js

```diff
@@ -0,0 +1,82 @@
+import { Component } from "react";
+import axios from "axios";
+
+import "./People.css"
+
+class People extends Component {
+  state = {
+    input: "",
+    personSearchArray: [],
+    hasSearched: false,
+  };
+
+  handleChange = (e) => {
+    this.setState({ input: e.target.value });
+    this.setState({ hasSearched: true });
```

Comment:
These setStates should be one call. 
this.setState({input: e.target.value, hasSearched: true})